multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -375,7 +375,7 @@ def object_detection(presets: str = DEFAULT):
         "optimization.top_k_average_method": "best",
         "optimization.warmup_steps": 0.0,
         "optimization.patience": 10,
-        "optimization.val_metric": "map",
+        # "optimization.val_metric": "map",
```

Comment:
why we comment out this?

Reply:
Because this PR changes the design of customizing validation metric. `optimization.val_metric` has been removed from the default config.

Reply:
Removed the commented line.